apps/easypid/src/features/wallet/FunkeCredentialDetailScreen.tsx

```diff
@@ -33,12 +34,13 @@ export function FunkeCredentialDetailScreen() {
   const { withHaptics } = useHaptics()
 
   const { credential } = useCredentialForDisplayById(params.id)
+  const [isDevelopmentModeEnabled] = useDevelopmentMode()
   const [isSheetOpen, setIsSheetOpen] = useState(false)
 
   useHeaderRightAction({
     icon: <HeroIcons.Trash />,
     onPress: withHaptics(() => setIsSheetOpen(true)),
-    renderCondition: credential?.category?.canDeleteCredential ?? true,
+    renderCondition: (isDevelopmentModeEnabled || credential?.category?.canDeleteCredential) ?? true,
```

Comment:
It's actually a requirement from ARF / EUDI to be able to remove the PID. So let's just enable it by default?

We should make sure to remove all instances though (so sd-jwt and mdoc)

Reply:
alrighty